### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by mahdizaabi**
Hey Malek! this the task 3

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,22 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """A method to retrieve one object:"""
+        if cls in classes.values():
+            a = self.all(cls)
```
Comment (malek10xdev) [inline models/engine/db_storage.py]: hey could u rename variable ```a``` to something relevant to what the function do.
naming examples: ```objects```,

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -74,3 +74,22 @@ def reload(self):
...
+            for k, v in a.items():
```
Comment (malek10xdev) [inline models/engine/db_storage.py]: could u rename k, v to ```key, value``` or ```key, object```

File: models/engine/db_storage.py [hunk 3]
```diff
@@ -74,3 +74,22 @@ def reload(self):
...
+                if v.id == id:
+                    return v
+        return None
+
+    def count(self, cls=None):
+        """count the number of objects in storage:"""
+        if cls is None:
+            count = 0
+            for c in classes.values():
```
Comment (maleksal) [inline models/engine/db_storage.py:91]: rename ```c``` to something relevant

**Review by malek10xdev — COMMENTED** [review]
Hi , could u fix the naming of variables (a, k,v and c)

**Comment by malek10xdev** [comment]
Code documentation and unittest looks good, thank u

File: models/engine/db_storage.py (hunk 1 above)
Comment (mahdizaabi) [reply models/engine/db_storage.py]: ok

File: models/engine/db_storage.py (hunk 3 above)
Comment (malek10xdev) [inline models/engine/db_storage.py]: rename c to something relevant

**Review by malek10xdev — CHANGES_REQUESTED** [review]
fix some naming conventions

**Review by malek10xdev — CHANGES_REQUESTED** [review]
fix some naming conventions

**Review by malek10xdev — APPROVED** [review]

**Pull request merged**